### DISCUSSION TITLE
GraphQL API Feature Request: Please add `items` to `ProjectV2View`

### DISCUSSION THREAD

**Discussion by MarcLapierre** [community/community#41776] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

There's currently no way to filter `items` on the server side within `ProjectV2`.

In order to avoid having to query for all items every time, a low-lift way to do simple filtering would be to allow `ProjectV2View` expose the `items` visible in that view via the API.

**Comment by khacminh** [comment]
+1 for this request. Time flies and my projects having thousand of issues. It's waste of resources when we have to query the issues over and over again

**Comment by rubengalle** [comment]
+1 to this old request or did there anything happen, to get server side filtered e.g. all issues in the current iteration?
